az-core/src/main/java/azkaban/Constants.java

```diff
@@ -291,9 +291,14 @@ public static class ConfigurationKeys {
     public static final String HISTORY_SERVER_JOB_URL = "azkaban.server.external.history_server_job_url";
     public static final String SPARK_HISTORY_SERVER_JOB_URL = "azkaban.server.external.spark_history_server_job_url";
 
-    // Configures the Kafka appender for logging user jobs, specified for the exec server
-    public static final String AZKABAN_SERVER_LOGGING_KAFKA_BROKERLIST = "azkaban.server.logging.kafka.brokerList";
-    public static final String AZKABAN_SERVER_LOGGING_KAFKA_TOPIC = "azkaban.server.logging.kafka.topic";
+    // Configures the Kafka appender for logging user jobs and flows.
+    public static final String AZKABAN_LOGGING_KAFKA_ENABLED = "azkaban.logging.kafka.enabled";
+    public static final String AZKABAN_LOGGING_KAFKA_CLASS_PARAM = "azkaban.logging.kafka.class";
+    public static final String AZKABAN_LOGGING_KAFKA_BROKERS = "azkaban.logging.kafka.brokers";
```

Comment:
Is this broker different from event reporter broker?

Reply:
The meaning is the same but values can be the same or different. (in our case, values are different)

Reply:
Thanks, just a little suggestion, we could consider reusing the broker and schema registry in the azkaban config.

Reply:
Yep, that makes sense ^. But actually we are using different broker for event reporter and for logging